NaviGator/mission_control/navigator_launch/launch/perception/classifier.launch

```diff
@@ -3,18 +3,22 @@
 
     <arg name="weights_model1" default="$(find yolov7_ros)/src/mil_weights/robotx.pt" />
     <arg name="weights_model2" default="$(find yolov7_ros)/src/mil_weights/robotx.pt" />
+    <arg name="weights_model3" default="$(find yolov7_ros)/src/mil_weights/stc.pt" />
     <arg name="data_yaml_model1" default="$(find yolov7_ros)/src/data/vrx_buoys.yaml" />
     <arg name="data_yaml_model2" default="$(find yolov7_ros)/src/data/vrx_buoys.yaml" />
+    <arg name="data_yaml_model3" default="$(find yolov7_ros)/src/data/stc.yaml" />
     <arg name="main_image_topic" default="/camera/front/left/image_color"/>
     <arg name="image_topic_model1" default="/yolov7/model1" />
     <arg name="image_topic_model2" default="/yolov7/model2" />
     <arg name="out_topic_model1" default="detections_model1" />
     <arg name="out_topic_model2" default="detections_model2" />
-    <arg name="conf_thresh" default="0.75" />
+    <arg name="out_topic_model3" default="stc_detections_model" />
+    <arg name="conf_thresh" default="0.25" />
     <arg name="image_size" default="640" />
-    <arg name="device" default="cuda" />
-    <arg name="use_yolo_model1" default="True"/>
-    <arg name="use_yolo_model2" default="True"/>
+    <arg name="device" default="cpu" />
```

Comment:
Should the device be set to `cuda` over `cpu` for better performance?

Reply:
we agree on cpu